hilda/snippets/macho.py

```diff
@@ -0,0 +1,91 @@
+import lldb
+from construct import *
+
+from hilda.symbol import SymbolFormatField
+
+LOAD_COMMAND_TYPE = Enum(Int32ul,
+                         LC_LOAD_DYLIB=0xc,
+                         LC_SEGMENT_64=0x19)
+
+
+class MachOStructFactory(object):
```

Comment:
i don't really see the use of this class since it doesn't do a factory. move all structs to be just flat out as global module structs. it's more readable this way